src/dbnode/storage/limits/query_limits.go

```diff
@@ -172,7 +171,9 @@ func newLookbackLimitMetrics(
 		total:           scope.Counter(fmt.Sprintf("total-%s", name)),
 		exceeded:        scope.Tagged(map[string]string{"limit": name}).Counter("exceeded"),
 
-		sourceLogger: sourceLoggerBuilder.NewSourceLogger(name, iopts),
+		// nb: no need to provide query-limits subscope to source logger,
+		// as it's not directly related to limits.
+		sourceLogger: sourceLoggerBuilder.NewSourceLogger(name, instrumentOpts),
```

Comment:
should this be a dependency instead that is passed in?

Reply:
name is still a dependency of source logger, so it makes sense in here